src/pages/authenticate/identity-selector/IdentitySelector2.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
```

Comment:
Why do we have a `IdentitySelector2` file?

Reply:
Forgot to delete it 🤦‍♂️